content/academy/apify_platform/publishing_your_actor/monetizing-your-actor.md

```diff
@@ -0,0 +1,88 @@
+﻿---
+title: Monetizing your actor
+description: Learn how you can use Apify to monetize your web scraping and automation projects.
+menuWeight: 1
+category: courses
+paths:
+  - monetizing-your-actor
```

Comment:
broken path should be apify_platform/publishing_your_actor/monetizing_your_actor

Btw, the MD files should be `snake_cased` instead of `using-dashes`